node/testing/src/bench.rs

```diff
@@ -95,9 +95,9 @@ pub fn drop_system_cache() {
 		return;
 	}
 
-	std::process::Command::new("sync")
-		.output()
-		.expect("Failed to execute system cache clear");
+	// std::process::Command::new("sync")
```

Comment:
Remove this change too.

Reply:
I'll remove all the not required changes ,